apps/test/unit/code-studio/components/progress/UnitCalendarButtonTest.jsx

```diff
@@ -0,0 +1,62 @@
+import {expect} from '../../../../util/reconfiguredChai';
+import React from 'react';
+import {shallow} from 'enzyme';
+import UnitCalendarButton from '@cdo/apps/code-studio/components/progress/UnitCalendarButton';
+import UnitCalendarDialog from '@cdo/apps/code-studio/components/progress/UnitCalendarDialog';
+
+const testLessons = [
```

Comment:
I think you used this test data in the storybook entry as well. It would be good to define it in one place and then share it between the files that need the sample data. Example: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/lib/levelbuilder/lesson-editor/activitiesTestData.js